data/hai/hai missions.txt

```diff
@@ -1658,3 +1658,123 @@ mission "Strider notification"
+			choice
+				`	"Alright then, I can take you there."`
+				`	"I'm not going that way. You'll need to find some other captain."`
+					decline
+			label accept
+				`	"Excellent! I'll let the crew know where to find you."`
+					accept
+	on visit
+		dialog phrase "generic cargo and passenger on visit"
+	on complete
+		event "hai film crew" 30
+		payment 100000 50
+		conversation
+			`You help your passengers unload all of their film equipment once you arrive on <planet>. The props seem to get more and more elaborate as they are unpacked from your ship. You note the presence of countless rubber limbs, mechanical monsters, and cans of fake blood being carried onto the set. Once you are finished, Likeet approaches you.`
+			choice
+				`	"What's the name of your movie, anyways?"`
+				`	"What's your movie about, anyways?"`
+				`	"I better get going, then."`
+					goto going
+			`	"I don't want to spoil anything, but it's a psychological horror film that explores the nature of human isolation, and the shadow that our own mortality casts over our lives." She continues to elaborate on more philosophical themes that seemingly have less and less to do with the actual film, until she notices that your eyes have long since glazed over.`
+			`	"Well, anyways, thanks for your help, Captain. I'll contact you in a month once we're finished," says Likeet.`
+				accept
+			label going
+			`	"Thanks for your help, Captain. I'll contact you in a month once we're finished," says Likeet.`
+				accept
+
+
+
+event "hai film crew"
+
+
+
+mission "Hai Film Crew [2]"
+	landing
+	name "Hai Film Crew"
+	description "Go to <destination> to pick up the film crew you transported there."
+	source
+		government "Hai" "Hai (Unfettered)" "Republic" "Syndicate" "Free Worlds" "Pirate" "Neutral" "Independent"
+	destination "Darkrest"
+	to offer
+		has "event: hai film crew"
+	on offer
+		dialog
+			`Upon landing your ship, you are startled by a vaguely familiar voice coming from your intercom. You soon recognize it as Likeet, the movie director that you brought to Darkrest a month ago.`
+			`	"Captain <last>! Good to see you again! We've finished filming here and need a ride off this place. I'll see you soon." Before you can say anything, she hangs up.`
+				accept
+
+
+
+mission "Hai Film Crew [3]"
+	minor
+	name "Hai Film Crew"
+	description "Take Likeet's film crew home to their studio on <destination>, so that they can process their film. Payment is <payment>."
+	cargo "film equipment" 30
+	passengers 80
+	source "Darkrest"
+	destination
+		attributes hai urban
+	to offer
+		has "Hai Film Crew [2]: done"
+	on offer
+		conversation
+			`The first thing you notice when you land on Darkrest is the mess. It looks as if a tornado has touched down on the set, and you don't remember Likeet selling the movie as being about natural disasters.`
```

Comment:
```suggestion
			`The first thing you notice when you land on Darkrest is the mess. It looks as if a tornado has touched down on the set, and you don't remember Likeet talking about natural disasters when she described the movie.`
```

"Selling the movie" is a bit of a confusing phrase, and if you don't know what to expect or don't have an idiomatic knowledge of English you could get really lost. After all, you haven't been buying tickets or copies of the movie from her... have you?